dense_vector/README.md

```diff
@@ -29,4 +29,5 @@ This track accepts the following parameters with Rally 0.8.0+ using `--track-par
 * `bulk_size` (default: 5000)
 * `bulk_indexing_clients` (default: 1): Number of clients that issue bulk indexing requests.
 * `index_settings`: A list of index settings. Index settings defined elsewhere (e.g. `number_of_replicas`) need to be overridden explicitly.
+* `max_num_segments`: The number of segments to target when doing a force merge (default: -1)
```

Comment:
Should the default value be `1` instead of `-1`, to avoid impacting the current stateful nightly benchmarks? Otherwise, we will need to set the parameter in the nightlies to avoid impacting benchmarking results.

Reply:
The current parameter in the nightlies is unset, which is the same as `-1`.  This has the same effect as passing `1` for the volume of data we use in nightly benchmarks.  It might be worth changing nightlies to pass `1` explicitly here, because if for some reason one of the benchmarks ends up with more than 5Gb of data in it you may end up with more than one segment after force merging.

Reply:
Right, thank you. This looks good for pmc, percolator, and nested. dense_vector and geopointshape both have `max-num-segments: 1`. Can we use `1` for those defaults?

Reply:
done!